tests/orquestra/integrations/qiskit/conversions/pauli_conversions_test.py

```diff
@@ -51,11 +49,9 @@ def test_paulisum_to_qiskitpauli():
 
     qiskit_op = qubitop_to_qiskitpauli(pauli_term)
 
-    ground_truth = (
-        PauliOp(Pauli("XZX"), 0.5) + PauliOp(Pauli("YZY"), 0.5)
-    ).to_pauli_op()
+    truth = SparsePauliOp.from_list([("XZX", 0.5), ("YZY", 0.5)])
```

Comment:
Question: Why not use the standard constructor here. [link](https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.SparsePauliOp#:~:text=still%20ordered%20lexicographically.-,Example,Here%20is%20an%20example%20of%20how%20to%20use%20SparsePauliOp%20argsort.,-import%20numpy%20as)